### DISCUSSION TITLE
Force camera to use fossify gallery

### DISCUSSION THREAD

**Discussion by yaxunil2** [FossifyOrg/General-Discussion#231] [Q&A]
on a Xiaomi redmi note 13 After taking a photo and then clicking on the photo thumbnail the phone opens the native gallery not fossify gallery? Even though I set fossify gallery as the 'default' gallery?? The native gallery can not be deleted and if I disable it, A notification to restore permission to it opens when I click the thumbnail?? Any body know how to resolve that?

**Comment by Aga-C** [comment]
Please create a discussion in [Gallery](https://github.com/FossifyOrg/Gallery/discussions).
